system/jlib/jstatcodes.h

```diff
@@ -251,4 +251,5 @@ enum StatisticKind
 };
 constexpr StatisticKind operator |(StatisticKind l, StatisticKind r) { return (StatisticKind)((unsigned)l | (unsigned)r); }
 
+typedef __int64 stat_t;
```

Comment:
This should be unsigned.

Reply:
Some statistics (such as SkewMinRowsProcessed) are negative.